R/infrastructure.R

```diff
@@ -301,30 +322,46 @@ use_package_doc <- function(pkg = ".") {
 #'   "Depends", "Suggests", "Enhances", or "LinkingTo" (or unique abbreviation)
 #' @param pkg package description, can be path or package name. See
 #'   \code{\link{as.package}} for more information.
+#' @param version Value indicating whether \code{package}'s version should be included
+#'   in \code{pkg}'s DESCRIPTION. Either \code{NULL}, \code{TRUE}, or a valid version
+#'   string. If \code{version} is \code{TRUE}, \code{package}'s current version
+#'   will be used (see examples). If \code{version} is a string that is not a
```

Comment:
I think you can eliminate the sentence describing what happens with invalid version specs